py2many/clike.py

```diff
@@ -164,10 +164,13 @@ def _cast(self, name: str, to) -> str:
     def _slice_value(self, node: ast.Subscript):
         # 3.9 compatibility shim
         if sys.version_info < (3, 9, 0):
-            if not isinstance(node.slice, ast.Index):
-                raise AstNotImplementedError("Advanced Slicing not supported", node)
-            slice_value = node.slice.value
+            if isinstance(node.slice, ast.Index):
+                slice_value = node.slice.value
+            else:
+                slice_value = node.slice
         else:
+            if isinstance(node.slice, ast.Slice):
```

Comment:
Left a comment on https://github.com/adsharma/py2many/issues/422 on this topic.

Reply:
This change here is only to make v0.3.0 consistent across Python versions.  I expect this will be supported at some stage, but it isnt worth holding up v0.3.0 to implement it.